documentation/requirements.md

```diff
@@ -0,0 +1,75 @@
+# Build requirements
+This file lists the build requirements for different BoB robotics modules and sample projects.
+
+## Compiler
+On Linux, the earliest supported version of gcc is 4.9. For Windows, Visual Studio 2015 or newer is required.
+
+## Environment variables
+The ``BOB_ROBOTICS_PATH`` environment variable is used to help programs built with BoB robotics modules to find where this repository is on your machine. (Currently this is only needed so that the panoramic unwrapping code can find [the unwrapping parameters for different types of camera](../imgproc/unwrapparams).)
+
```

Comment:
If you could add words to these effect it would also be much-appreciated:

> It should also be passed to genn-buildmodel using the -i command line argument when building networks using the models in the genn_models folder i.e. ``genn-buildmodel.sh -i $BOB_ROBOTICS_PATH`` on *nix or ``genn-buildmodel.bat -i %BOB_ROBOTICS_PATH`` on Windows